src/evaluators/utility/PHAL_ComputeBasisFunctionsSide.hpp

```diff
@@ -64,20 +64,38 @@ class ComputeBasisFunctionsSide : public PHX::EvaluatorWithBaseImpl<Traits>,
 
   // Output:
   //! Basis Functions and other quantities at quadrature points
-  PHX::MDField<MeshScalarT,Cell,Side,QuadPoint>           metric_det;
-  PHX::MDField<MeshScalarT,Cell,Side,QuadPoint,Dim,Dim>   tangents;
-  PHX::MDField<MeshScalarT,Cell,Side,QuadPoint,Dim,Dim>   metric;
-  PHX::MDField<MeshScalarT,Cell,Side,QuadPoint>           w_measure;
-  PHX::MDField<MeshScalarT,Cell,Side,QuadPoint,Dim,Dim>   inv_metric;
-  PHX::MDField<RealType,Cell,Side,Node,QuadPoint>         BF;
-  PHX::MDField<MeshScalarT,Cell,Side,Node,QuadPoint,Dim>  GradBF;
+  PHX::MDField<MeshScalarT>                               metric_det;
+  PHX::MDField<MeshScalarT>                               tangents;
+  PHX::MDField<MeshScalarT>                               metric;
+  PHX::MDField<MeshScalarT>                               w_measure;
+  PHX::MDField<MeshScalarT>                               inv_metric;
+  PHX::MDField<RealType>                                  BF;
+  PHX::MDField<MeshScalarT>                               GradBF;
   PHX::MDField<MeshScalarT,Cell,Side,QuadPoint,Dim>       normals;
 
-  std::vector<std::vector<int> > sideNodes;
-  std::vector<Kokkos::DynRankView<int, PHX::Device>> cellsOnSides;
-  std::vector<int> numCellsOnSide;
+  //std::vector<Kokkos::DynRankView<int, PHX::Device>> cellsOnSides;
+  //std::vector<int> numCellsOnSide;
```

Comment:
Remove all cell on sides comments if we're not going to use them anymore. There's some in the definition too.

Reply:
I think these comments still need to be removed.

Reply:
Woops, I thought I got all of them. I'll do another pass to make sure they all get removed.